hubploy/auth.py

```diff
@@ -77,7 +77,7 @@ def registry_auth_gcloud(deployment, project, service_key):
     yield
 
 
-def registry_auth_aws(deployment, project, zone, service_key=None, role_arn=None):
```

Comment:
Without normalization elsewhere, this will fail if the hubploy.yaml file specifies 'project' and 'zone' right?

Reply:
I don't think it will. These are just what `registry_auth_aws()` will use the variables as, right? They are passed in as 

```
registry_auth_aws(deployment, **registry['aws'])
```

so I don't think it can break. It just expects them in the order `[project, zone, service_key, ...]`. That's what I meant about being automatically backward compatible.

I can check that this is actually true though, I have a Z2JH thing I was going to try to stand up this week.

Reply:
To be more specific, I bet if you even specified completely different options where `project` and `zone` are in `hubploy.yaml`, this function wouldn't notice until it tried to use the string it built to ping the registry.

Reply:
Ah, I am incorrect. It does give me 

```
TypeError: cluster_auth_aws() got an unexpected keyword argument 'account_id'
```

when I changed the entry in `hubploy.yaml` but not in `auth.py`. So I'll change it in both places, and it won't be backwards-compatible.